anaconda_anon_usage/tokens.py

```diff
@@ -76,16 +118,24 @@ def token_string(prefix=None, enabled=True):
     appended to the conda user agent.
     """
     parts = ["aau/" + __version__]
-    if enabled:
+    if enabled or system_token():
+        if not enabled:
+            _debug("anaconda_anon_usage enabled by system token")
         values = all_tokens(prefix)
         if values.client:
             parts.append("c/" + values.client)
         if values.session:
             parts.append("s/" + values.session)
         if values.environment:
             parts.append("e/" + values.environment)
+        if values.system:
+            parts.append("o/" + values.system)
```

Comment:
Watch out we'll soon run out of characters 😄

Reply:
lol, to be fair, we already use `o/` in anaconda-ident